onnxruntime/test/providers/qnn/resize_test.cc

```diff
@@ -188,7 +186,7 @@ static void RunQDQResizeOpTest(const TestInputDef<float>& input_def,
 
 // Upsample that uses "round_prefer_floor" as the "nearest_mode".
 // coordinate_transformation_mode: "half_pixel"
-TEST_F(QnnCPUBackendTests, DISABLED_ResizeUpsampleNearestHalfPixel_rpf) {
+TEST_F(QnnCPUBackendTests, ResizeUpsampleNearestHalfPixel_rpf) {
```

Comment:
I don't think we should enable these tests yet. It doesn't make sense that QNN's ResizeNearestNeighbor is equivalent to ONNX's Resize operator with multiple different rounding modes (round_prefer_floor, round_prefer_ceil). Said another way, if A != B, then it would be a contradiction for C to equal both A and B.

I think we should ask QC (or look at the latest qnn-onnx-converter) what the behavior is supposed to be. I suspect that these tests may not be exercising the difference in rounding modes. We may need to write tests with explicit inputs (non-random) that properly check the rounding behavior.